### PR TITLE
Harmony app proto-prototype

### PR DESCRIPTION
**Pull request by jeffbaumes**
This also does some reorg of the main library which still needs some work.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "Visualization components for the web",
   "main": "index.js",
   "scripts": {
-    "build": "webpack"
+    "build": "webpack",
+    "build-harmony": "cd app/harmony && webpack"
```
Comment (ronichoudhury) [inline package.json]: Can we use ``build:harmony`` instead? I feel like the colon "scopes" the build (or other) subcommands better. (If we're ok with the change, it needs to be changed in the Harmony README as well.)

**Comment by ronichoudhury** [comment]
Aside from the line item above, LGTM.

I'm going to reorg the demo app in a separate PR.

**Comment by jeffbaumes** [comment]
I've updated the script name to use colon.

**Comment by ronichoudhury** [comment]
If you update the README as well I think this is ready to merge.

**Comment by jeffbaumes** [comment]
Done.

**Comment by ronichoudhury** [comment]
Should we follow the "PR owner presses the launch button" protocol?

**Comment by jeffbaumes** [comment]
Yes please, but I'm looking for a LGTM - "i think this is ready to merge" is not official enough for my taste.

**Comment by ronichoudhury** [comment]
Oh, I thought my earlier "LGTM" would suffice :)

*ahem*... LGTM!

**Comment by jeffbaumes** [comment]
Hooray! You are right you did give a provisional LGTM that should have sufficed.

**Pull request merged**